src/webhook/setNews.js

```diff
@@ -34,29 +33,24 @@ const createQueueNews = async (req, res) => {
             id_feed,
             user_id,
             topics,
-            duration_feed,
+            duration_feed
           },
           options
         );
-        return successResponse(
-          res,
-          `success created news with job id : ${getJob.id}`,
-          bodyData
-        );
-      } else {
-        return errorResponse(res, 'url is invalid', 400);
-        // throw new Error("url is invalid");
+        return successResponse(res, `success created news with job id : ${getJob.id}`, bodyData);
       }
+      return errorResponse(res, 'url is invalid', 400);
+      // throw new Error("url is invalid");
     } catch (error) {
       console.error(error);
       return errorResponse(res, error.toString(), 500);
     }
   } else {
     console.log('sukses tanpa data');
-    return successResponse(res, "ok", "Job running");
+    return successResponse(res, 'ok', 'Job running');
   }
 };
 
 module.exports = {
-  createQueueNews,
+  createQueueNews
 };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [28-36]

The `newsQueue.add` function is called without any error handling. If this function throws an error, it will not be caught and could potentially crash the application. Consider wrapping this call in a try-catch block.
```diff
  if (checkIfValidURL(bodyData)) {
+   try {
      const getJob = await newsQueue.add(
        id_feed,
        user_id,
        topics,
        duration_feed
      );
+   } catch (error) {
+     console.error(error);
+     return errorResponse(res, error.toString(), 500);
+   }
    return successResponse(res, `success created news with job id : ${getJob.id}`, bodyData);
  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->